addons/stock_dropshipping/models/sale.py

```diff
@@ -1,7 +1,42 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields, _
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    purchase_order_count = fields.Integer(compute='_compute_purchase_order_count',
+                                          groups='purchase.group_purchase_user')
+
+    @api.depends('order_line.purchase_line_ids.order_id')
+    def _compute_purchase_order_count(self):
+        for sale in self:
+            sale.purchase_order_count = len(sale._get_purchase_order())
+
+    def _get_purchase_order(self):
+        return self.order_line.purchase_line_ids.order_id
```

Comment:
here https://github.com/odoo/odoo/pull/46013#issuecomment-590202024 i told you to use `_get_purchase_orderS` from the linked rev , doesn t it work?